models/agent_model.go

```diff
@@ -631,6 +638,18 @@ func (s Agent) TemplateDelimiters(svc *Service) *DelimiterPair {
 	return &tdp
 }
 
+func (s *Agent) buildWebConfigFile(password string) string {
+	buf, err := bcrypt.GenerateFromPassword([]byte(password), 14)
```

Comment:
`14` is magic var

Reply:
currently can move it to some local const, but maybe we should introduce global configurable value for that?

Reply:
I think moving to const is good for now